pages/clubs-toolkit.jsx

```diff
@@ -46,6 +46,17 @@ var ToolkitList = React.createClass({
 
 var ToolkitListItem = React.createClass({
   render: function() {
+    if (this.props.headerHref) {
+      return (
+        <li>
+          <p>
+            <span className="italic"><a href={this.props.headerHref}>{this.props.header}</a> </span>
+            <span>{this.props.children}</span>
```

Comment:
It seems like we can only pass strings to `header`? 
I tried simple things like `<ToolkitListItem header="<b>Discussion Forum</b>">` but the tags were seen as strings instead (so it displays `<b>Discussion Forum.</b>`, not **Discussion Forum.**).  Maybe there are some React methods or syntax that I'm not aware of...?